client/css/style.scss

```diff
@@ -24,4 +24,5 @@
 @import 'update_status';
 @import 'hangout_card';
 @import 'hangout_board';
-@import 'study_groups';
\ No newline at end of file
+@import 'study_groups';
+@import 'variables'; // global vars to be added to this
```

Comment:
Let's move the `@import 'variables';` up to the very top -- before `@import 'colors';`